### ISSUE TITLE
Accessibility finding: Some text doesn't resize when font size is set to "Very Large"

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#139072]
### Checklist item

ADE-004	Font size can be set to "Very Large" or equivalent without breaking the page

### Severity level

3, Medium. Should be fixed in 1-3 sprints post-launch.

### Page(s)/url(s)

https://staging.va.gov/manage-dependents/add-remove-form-21-686c-674/confirmation
https://staging.va.gov/my-va/submissions/9093f30e-0d09-43ff-a13f-087847063071?postLogin=true

### Details

"Some text is "naked," contained in a `div` or `span` without any semantic tags like `p`.

When a user sets their font size to "Very Large," that text doesn't resize along with the rest of the page.

This text includes:

CONFIRMATION PAGE: 
- All text in the "Save a copy of your form" section
- "Use the Claim Status Tool..." link
- "Start a new dependency claim" link
- "Contact us online through Ask VA" link

FORM VIEWER PAGE:
- OMB Version...
- Signed electronically...
- Not answered
- Signee signed with...

Example: compare the size of "By common law" to "Not answered"

<img width="398" height="484" alt="Image" src="https://github.com/user-attachments/assets/45a3cc43-3b6e-49d0-83c5-595320f30c1b" />

### Reproduction steps

In Chrome, use the Settings to change font size to "Very Large." Then go to the page and observe the text that doesn't resize.

### Proposed solution or next steps

Put the text in a `p` tag.
